### DISCUSSION TITLE
Session root path configuration

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAILabel#1485] [Q&A]
Hello, 

when starting a monailabel server, is there a way to define the session root path?

I would like to be able to look at the session data whenever something goes wrong to actually have a look at the input/output data.

**Comment by SachidanandAlle** [comment]
Which session? On the server side.. in that case it will be under workspace or user home by default..

You can check config.py for more details

**Reply by che85** [reply]
On the server side. I am running monailabel inside a docker container so there would not be an easy way to get to the data unless I copy from container to host or create a direct mapping.

**Comment by SachidanandAlle** [comment]
https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/config.py#L88

you set your own path.. (docker workspace which you mount)
`export MONAI_LABEL_SESSION_PATH=/myworkspace/sessions`
